core/vm/gas.go

```diff
@@ -13,45 +14,10 @@ var (
 	GasSlowStep    = big.NewInt(10)
 	GasExtStep     = big.NewInt(20)
 
-	GasStorageGet        = big.NewInt(50)
```

Comment:
You could have saved yourself a lot of trouble by changing **just** the assignment :-)